PIL/Image.py

```diff
@@ -1122,6 +1186,7 @@ def filter(self, filter):
         return merge(self.mode, ims)
 
     def getbands(self):
+        # type: () -> Tuple[Any]
```

Comment:
This was due to the same point I raised elsewhere regarding Tuples of unknown length.